python/paddle/fluid/dygraph/dygraph_to_static/loop_transformer.py

```diff
@@ -249,6 +253,18 @@ def visit_While(self, node):
         self.generic_visit(node)
         self.current_loop.pop()
 
+    def visit_Call(self, node):
+        # Store type var names such as "isinstance(x, some_type_names)" and
+        # Remove them later
+        if isinstance(node.func, gast.Name) and node.func.id == 'isinstance':
+            type_node = node.args[1]
+            if isinstance(type_node, gast.Tuple):
```

Comment:
Is node.args[1] also `gast.Tuple` in usage `isinstance(var, [type1, type1])` ? or its type is `gast.List`?